test/presenters/return-requirements/check/returns-requirements.presenter.test.js

```diff
@@ -141,6 +143,50 @@ describe('Return Requirements presenter', () => {
           })
         })
       })
+
+      describe('and the agreement exceptions', () => {
+        describe('is \'none\'', () => {
```

Comment:
NIT - [Our new convention](https://github.com/DEFRA/water-abstraction-team/blob/main/coding_conventions.md#unit-test-descriptive-text)

```suggestion
        describe('is "none"', () => {
```